app/assets/stylesheets/application.css.scss

```diff
@@ -242,3 +242,7 @@ div#bp_ad_end {
    border-radius: 3px;
    border:1px solid black;
  }
+
+.review-text {
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.